docs/Ecosystem.md

```diff
@@ -140,3 +140,4 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-ws`](https://github.com/gj/fastify-ws) WebSocket integration for Fastify — with support for WebSocket lifecycle hooks instead of a single handler function. Built upon [ws](https://github.com/websockets/ws) and [uws](https://github.com/uNetworking/bindings/tree/master/nodejs).
 - [`k-fastify-gateway`](https://github.com/jkyberneees/fastify-gateway) API Gateway plugin for `fastify`, a low footprint implementation that uses the `fastify-reply-from` HTTP proxy library.
 - [`i18next-http-middleware`](https://github.com/i18next/i18next-http-middleware#fastify-usage) An [i18next](https://www.i18next.com) based i18n (internationalization) middleware to be used with Node.js web frameworks like express or `Fastify` and also for Deno.
+- [`fastify-now`](https://github.com/yonathan06/fastify-now) Structure your endpoints in a folder and load them dynamically with fastify
```

Comment:
Could you move your plugin in alphabetical order?

Moreover, I would improve your plugin:
- add tests
- remove the `package-lock` because it is not released for modules and it could hide bad dependencies updates

Reply:
Thank you for the quick review
I did what you asked (reordered it alphabetically, removed `package-lock` and added tests)